.changeset/many-oranges-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": major
+---
+
+Remove `EditPageLayout`
```

Comment:
Please extend this changeset why this isn't necessary anymore and what devs should do instead.